x-pack/filebeat/module/salesforce/_meta/config.yml

```diff
@@ -0,0 +1,127 @@
+- module: salesforce
+
+  apex-rest:
+    enabled: false
+
+    # Oauth Client ID
+    # var.client_id: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+
+    # Oauth Client Secret
+    # var.client_secret: "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"

```

Comment:
@marc-gr Do you know if there is an option to set secrets in one place instead of per-config?